tf-build-tools/tf-maven-plugin/src/main/resources/tf/terraform-maven.properties

```diff
@@ -1,6 +1,6 @@
-distributionSite=https://releases.hashicorp.com
-releaseDir=terraform
+distributionSite=https://repo.dtcc.com
```

Comment:
please remove employer specific stuff from GitHub

Reply:
@Clayton7510  I changed it back to the public release site and directory, and pushed